compiler/rustc_feature/src/active.rs

```diff
@@ -414,6 +414,8 @@ declare_features! (
     (incomplete, generic_const_exprs, "1.56.0", Some(76560), None),
     /// Allows using `..X`, `..=X`, `...X`, and `X..` as a pattern.
     (active, half_open_range_patterns, "1.41.0", Some(67264), None),
```

Comment:
I suppose this line should be removed since `half_open_range_patterns` is stabilized

Reply:
Yes, it should have been, not sure why it's wasn't caught by CI. I must have re-add it when doing a rebase. I've opened https://github.com/rust-lang/rust/pull/102883 to fix it.

Reply:
> not sure why it's wasn't caught by CI.

Because there isn't test to check feature duplications with different statuses